apt/repositories.sls

```diff
@@ -26,8 +26,30 @@ debian-archive-keyring:
     - group: root
     - clean: {{ clean_sources_list_d }}
 
+
+
 {% for repo, args in repositories.iteritems() %}
-{%- set r_arch = '[arch=' ~ args.arch|join(',') ~ ']' if args.arch is defined else '' %}
+
+{% set r_opts = '' %}
+{%- set r_arch = 'arch=' ~ args.arch|join(',') if args.arch is defined else '' %}
+{% if args.opts is defined %}
+  {% if args.opts is string %}
+    {% set r_opts = args.opts %}
+  {% else %}
+    {% set r_opts_list = [] %}
+    {%- for k, v in args.opts.items() %}
+      {% do r_opts_list.append(' ' ~ k ~ '=' ~ v) %}
```

Comment:
As you're converting the hash into a list, you can skip the initial whitespace here, so it'd be
```python
 {% do r_opts_list.append(k ~ '=' ~ v) %}
```

Reply:
fixed, thanks